modules/api/pkg/api/v2/types.go

```diff
@@ -1713,6 +1713,7 @@ type ResourceQuota struct {
 	SubjectKind string `json:"subjectKind"`
 	// SubjectHumanReadableName contains the human-readable name for the subject(if applicable). Just filled as information in get/list.
 	SubjectHumanReadableName string              `json:"subjectHumanReadableName,omitempty"`
+	Default                  bool                `json:"default"`
```

Comment:
Do we have any other similar case in KKP codebase? If not I suggest renaming this field to `IsDefault`. Maybe it's just me, but IMO it makes field interpretation much clearer.